splinepy/io/gismo.py

```
@@ -11,6 +11,297 @@
+                for component_index, function_component_string in enumerate(
+                    function_string
+                )
+            ]
+
+        # Add comment if given
+        if comment is not None:
+            function_dict["comment"] = comment
+
+        self._blocks.append(function_dict)
+
+    def add_assembly_options(
```

Pylint (3.3.7)

[refactor] Too many arguments (12/5)

(R0913)